nursery/curses/setup.cfg

```diff
@@ -0,0 +1,58 @@
+[metadata]
+name = toga-curses
+description = A Curses backend for the Toga widget toolkit.
+url = https://beeware.org/project/projects/libraries/toga/
+project_urls =
+    Funding = https://beeware.org/contributing/membership/
+    Documentation = http://toga.readthedocs.io/en/latest/
+    Tracker = https://github.com/beeware/toga/issues
+    Source = https://github.com/beeware/toga
+author = Russell Keith-Magee
+author_email = russell@keith-magee.com
+maintainer = BeeWare Team
+maintainer_email = team@beeware.org
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Software Development
+    Topic :: Software Development :: User Interfaces
+    Topic :: Software Development :: Widget Sets
+license = New BSD
+license_file = LICENSE
+long-description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+keywords =
+    gui
+    widget
+    cross-platform
+    toga
+    desktop
+    console
+    curses
+
+[options]
+packages = find:
+python_requires = >= 3.5
+zip_safe = False
+
+[options.packages.find]
+include = toga_curses
+
+[flake8]
+exclude=\
+    .eggs/*,\
+    build/*
+max-complexity = 10
+max-line-length = 119
+ignore = E121,E123,E126,E226,E24,E704,W503,W504,C901
+
+[isort]
```

Comment:
Do not use *isort* until it gets to version 5.0.0. It has some major issues that needs to be fixed until then, including consistent handling of 3rd parties libraries in virtual environments. Look at timothycrosley/isort#1147

Reply:
That's an interesting bug, to be sure; but I'm not sure it's likely to affect us in practice. It seems to mostly affect packages with... dubiously names (like a top level "config") package. 

In practical terms: I've been using isort in production for years without any problems. On top of that, in this particular instance, isort isn't actually hooked into CI. I added the configuration so that we get "black compatible" import styling. This lets us run isort, get imports that are sorted and formatted cleanly, and audit the results before commiting. 

In truth, there's a couple of files (like the GTK and Winforms libs module) that we'll need to turn off isort, because they have very specific import ordering. So it's unlikely we're ever going to add isort to *CI* - the configuration is just here as a convenience.

Reply:
Perhaps this merits adding a comment next to the `[isort]` configuration, so that people can quickly get up-to-speed on how we use `isort` in this project.

Reply:
I had a lot of problems with that in Eddington. I still hold to my suggestion.